src/engine/strat_engine/backstore/setup.rs

```diff
@@ -53,14 +53,25 @@ pub fn find_all() -> StratisResult<HashMap<PoolUuid, HashMap<Device, PathBuf>>>
         enumerator.match_property("ID_FS_TYPE", "stratis")?;
         for devnode in enumerator
             .scan_devices()?
-            .filter(|dev| dev.is_initialized())
+            .filter(|dev| {
+                let initialized = dev.is_initialized();
+                if !initialized {
+                    warn!("Found a udev entry for a device identified as a Stratis device, but udev also identified it as uninitialized, discarded the device from the set of devices to process, for safety");
```

Comment:
The phrase "discarded the device" might not be the best choice here; it could be interpreted as "sending discards to the device", which I don't believe is happening in this context.